### PR TITLE
Replace the references to gitter by Discord

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ waiting to be read.
 For fastest answers, join our [Scala.js Gitter chat room](https://gitter.im/scala-js/scala-js) and ask your questions,
 or just chat with other Scala.js developers! There are currently over 500 users in the chat room.
 
+### [`#scala-js` on Discord](https://discord.com/invite/scala)
+
+The [official Scala discord](https://discord.com/invite/scala) has over 600 members.
+Join the `#scala-js` channel if you have any questions about Scala.JS!
```
Comment (sjrd) [inline community/index.md:22]:
```suggestion
Join the `#scala-js` channel if you have any questions about Scala.js!
```

File: community/index.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ waiting to be read.
 For fastest answers, join our [Scala.js Gitter chat room](https://gitter.im/scala-js/scala-js) and ask your questions,
 or just chat with other Scala.js developers! There are currently over 500 users in the chat room.
 
+### [`#scala-js` on Discord](https://discord.com/invite/scala)
+
+The [official Scala discord](https://discord.com/invite/scala) has over 600 members.
```
Comment (sjrd) [inline community/index.md:21]: That figure is going to get obsolete before we know it. I suggest not to include that.

**Review by sjrd — CHANGES_REQUESTED** [review]
Thanks. I have two minor comments.

File: community/index.md (hunk 2 above)
Comment (gzm0) [reply community/index.md:21]: I think it's here because the gitter channel above also has it :)

**Comment by gzm0** [comment]
QQ: Shall we *stop* recommending to use gitter? It seems odd to suggest two "official" chat channels.

**Comment by sjrd** [comment]
Yes, I think we can also stop sending people to gitter. Virtually the entire Scala community has abandoned gitter for discord.

**Comment by kubukoz** [comment]
I didn't do it here because I saw people still posting a couple days ago, but I'll adjust it to get rid of hitter!

File: community/index.md (hunk 2 above)
Comment (kubukoz) [reply community/index.md:21]: indeed, I was inspired by the above :)

**Comment by kubukoz** [comment]
Dropped gitter, changed casing on "Scala.js"

File: community/index.md (hunk 2 above)
Comment (kubukoz) [reply community/index.md]: gone now

File: community/index.md (hunk 1 above)
Comment (kubukoz) [reply community/index.md]: done as part of the latest commit

**Pull request merged**